File: app/src/main/java/com/example/pam_project/lists/lists/components/SpinnerActivity.java
package com.example.pam_project.lists.lists.components;

import android.app.Activity;
import android.view.View;
import android.widget.AdapterView;

public class SpinnerActivity extends Activity implements AdapterView.OnItemSelectedListener {

    public void onItemSelected(AdapterView<?> parent, View view,
                               int pos, long id) {
        // Maybe put category color on spinner for user to check if category is ok
    }

    public void onNothingSelected(AdapterView<?> parent) {
        // Complete if categories can disappear while on view
    }
}

File: app/src/main/java/com/example/pam_project/features/categories/spinner/SpinnerActivity.kt
package com.example.pam_project.features.categories.spinner

import android.app.Activity
import android.view.View
import android.widget.AdapterView
import android.widget.AdapterView.OnItemSelectedListener

class SpinnerActivity : Activity(), OnItemSelectedListener {
    override fun onItemSelected(parent: AdapterView<*>?, view: View,
                                pos: Int, id: Long) {
        // Maybe put category color on spinner for user to check if category is ok
    }

    override fun onNothingSelected(parent: AdapterView<*>?) {
        // Complete if categories can disappear while on view
    }
}